components/exploits/ExploitStatus.tsx

```diff
@@ -20,12 +20,16 @@ const ExploitStatus = ({ address, chainId, exploit, status }: Props) => {
   const icons = {
     affected: <ExclamationTriangleIcon className="h-12 w-12 text-red-500" />,
     safe: <CheckCircleIcon className="h-12 w-12 text-green-500" />,
+    previously_accessible: <ShieldCheckIcon className="h-12 w-12 text-green-500" />,
   };
 
   const addressesForChainId = exploit.addresses.filter((address) => address.chainId === chainId);
   const spenderSearch = addressesForChainId.map(({ address }) => address).join(',');
   const baseHref = `/address/${address}?chainId=${chainId}`;
-  const href = status === 'affected' ? `${baseHref}&spenderSearch=${spenderSearch}` : baseHref;
+  const href =
+    status === 'affected' || status === 'previously_accessible'
```

Comment:
If the status was previously affected I think we can just keep the `baseHref`, since there are no required actions for the user to take.